Content.IntegrationTests/Tests/Power/StationPowerTests.cs

```diff
@@ -0,0 +1,100 @@
+using System.Collections.Generic;
+using System.Linq;
+using Content.Server.GameTicking;
+using Content.Server.Maps;
+using Content.Server.NodeContainer;
+using Content.Server.Power.Components;
+using Content.Server.Power.NodeGroups;
+using Content.Server.Power.Pow3r;
+using Robust.Server.GameObjects;
+
+namespace Content.IntegrationTests.Tests.Power;
+
+[TestFixture]
+public sealed class StationPowerTests
+{
+    /// <summary>
+    /// How long the station should be able to survive on stored power if nothing is changed from round start.
+    /// </summary>
+    private const float MinimumPowerDurationSeconds = 10 * 60;
+
+    private static readonly string[] GameMaps =
```

Comment:
See if there's a better way of doing this that can be shared between multiple tests.